db/src/main/kotlin/edu/artic/db/Accessiblable.kt

```diff
@@ -0,0 +1,8 @@
+package edu.artic.db
+
+/**
+ * Used to ensure content that requires content description can use this interface
```

Comment:
Change comment to

    Used to provide a reasonable content description to accessibility services